port to sass

martinlindhe opened the pull request:
Hello

here is a SASS port, quickly converted from the LESS code, for use in a sass-based asset chain

/m

tristanlins commented:
@martinlindhe please see https://github.com/martinlindhe/flag-icon-css/pull/1 :-)

martinlindhe commented:
@tristanlins thanks! merged :)

lipis commented on sass/flag-icon-base.scss:
```diff
@@ -0,0 +1,31 @@
+      background-image: url(#{$flag-icon-css-path}#{$flag-icon-square-path}/#{$country}.svg);
+    }
+  }
+}
```
can you keep the same number of lines to have it one to one.. it will be easier for the conversations later.. and if @mixn flag-icon-background has to be first.. then change the order in the less file as well.. but either case I want them to have the same in terms of lines :)

lipis commented:
I refactored a bit.. https://github.com/lipis/flag-icon-css/blob/master/less/flag-icon-base.less

martinlindhe commented:
@lipis i'll look into this hopefully tonight

martinlindhe commented:
@lipis Hi again. I have updated my branch and it is now line-by line as close as it can be with the flag-icon-base.less.

I did originally reorder the .flag-icon properties alphabetically because it was suggested by scss-lint, but i reverted it to match your less version, and removed some empty lines. Hope this is fine to merge!

Thanks

/m

lipis commented:
Would you mind squashing the commits?

martinlindhe commented:
@lipis ive created a new pull request with a squashed commit. had troubles rebasing this branch because i merged with your master.
